server/routes/__tests__/money.spec.js

```diff
@@ -372,30 +325,13 @@ describe('Prisoner Money', () => {
     });
 
     it('allows the user to specify the month for transactions by passing a date', async () => {
-      const mockPrisonerInformationService = {
-        getTransactionsFor: jest.fn(() => ({
-          transactions: transactionApiResponse,
-          balances: balancesApiResponse,
-        })),
-      };
-
-      const mockedMoneyRouter = createMoneyRouter({
-        hubContentService: {},
-        offenderService: {},
-        prisonerInformationService: mockPrisonerInformationService,
-      });
-
-      app.use(setMockUser);
-      app.use('/money', mockedMoneyRouter);
-      app.use(consoleLogError);
+      const spy = jest.spyOn(prisonerInformationService, 'getTransactionsFor');
```

Comment:
maybe calling this `transactionsRequest` or something might be more descriptive than `spy`